docs/extensions/creating-extension.rst

```diff
@@ -53,3 +53,16 @@ to ensure your extension's extraordinary quality.
 .. _`Behat`: http://behat.org/en/latest/
 .. _`phpspec`: http://www.phpspec.net/en/stable/
 .. _`PHPUnit`: https://phpunit.de/
+
+5. Naming conventions
+---------------------
+
+Beside the way how you would like to create a plugin, (based on our skeleton or by your own) there are a few naming conventions that should be followed:
+
+ * Bundle class must have a `Plugin` suffix instead of `Bundle` in its name (e.g. InvoicePlugin).
+ * Bundle class must use the `Sylius\Bundle\CoreBundle\Application\SyliusPluginTrait`.
+ * The name of the extension in DependencyInjection folder must follow the regular Symfony rules (e.g. InvoiceExtension).
```

Comment:
I don't know how to say this more exactly. `Extension` file from DI folder(which has to have `Extension` prefix) shouldn't have `Plugin` inside its name. See this example:
Plugin name is AcmeExample**Plugin**
<img width="410" alt="Plugin" src="https://cloud.githubusercontent.com/assets/6213903/24493416/02ad334a-152f-11e7-8021-f6982cb474e4.png">
Extension name is AcmeExample**Extension**
<img width="558" alt="Extension" src="https://cloud.githubusercontent.com/assets/6213903/24493457/2af4e460-152f-11e7-965e-9115ffc83844.png">